detection_rules/main.py

```diff
@@ -293,9 +294,16 @@ def view_rule(ctx, rule_file, api_format):
     return rule
 
 
-def _export_rules(rules: RuleCollection, outfile: Path, downgrade_version: Optional[definitions.SemVer] = None,
-                  verbose=True, skip_unsupported=False, include_metadata: bool = False):
-    """Export rules into a consolidated ndjson file."""
+def _export_rules(
+    rules: RuleCollection,
+    outfile: Path,
+    downgrade_version: Optional[definitions.SemVer] = None,
```

Comment:
Any reason not to use semantic versioning library versus our custom one?

Reply:
Duplicating from main, no specific reason to use the custom one.